protobuff/archive.proto

```diff
@@ -228,30 +291,83 @@ message GetTransactionResponseV2 {
 
 service ArchiveService {
 
-  //Tick
-  rpc GetAllTickTransactionsV2(GetTickTransactionsRequestV2) returns (GetTickTransactionsResponseV2){
+  // Latest Tick
+  rpc GetLatestTickV2(google.protobuf.Empty) returns (GetLatestTickResponseV2){
+    option (google.api.http) = {
+      get: "/v2/tick/latest"
+    };
+  };
+
+  // General data - To be released at a later date
+  /*rpc GetTickV2(GetTickRequestV2) returns (GetTickResponseV2) {
+    option (google.api.http) = {
+      get: "/v2/tick/{tick_number}"
+    };
+  };*/
+
+  // Quorum data
+  rpc GetTickQuorumDataV2(GetTickRequestV2) returns (GetQuorumTickDataResponse) { // Uses V1 response
+    option (google.api.http) = {
+      get: "/v2/tick/{tick_number}/quorum-data"
+    };
+  };
+
+  // Chain hash
+  rpc GetTickChainHashV2(GetTickRequestV2) returns (GetChainHashResponse) { // Uses V1 response
+    option (google.api.http) = {
+      get: "/v2/tick/{tick_number}/hash"
+    };
+  }
+
+  // Store hash
+  rpc GetTickStoreHashV2(GetTickRequestV2) returns (GetChainHashResponse) { // Uses V1 response
+    option (google.api.http) = {
+      get: "/v2/tick/{tick_number}/store-hash"
+    };
+  }
+
+
+  // Tick transactions
+  rpc GetAllTickTransactionsV2(GetTickRequestV2) returns (GetTickTransactionsResponseV2){
     option (google.api.http) = {
       get: "/v2/tick/{tick_number}/transactions"
     };
   };
-  rpc GetTransferTickTransactionsV2(GetTickTransactionsRequestV2) returns (GetTickTransactionsResponseV2){
+  rpc GetTransferTickTransactionsV2(GetTickRequestV2) returns (GetTickTransactionsResponseV2){
     option (google.api.http) = {
       get: "/v2/tick/{tick_number}/transactions/transfers"
     };
   };
-  rpc GetApprovedTickTransactionsV2(GetTickTransactionsRequestV2) returns (GetTickTransactionsResponseV2){
+  rpc GetApprovedTickTransactionsV2(GetTickRequestV2) returns (GetTickTransactionsResponseV2){
     option (google.api.http) = {
       get: "/v2/tick/{tick_number}/transactions/transfers/approved"
```

Comment:
this endpoint along with others should use `transfers` and `approved` as query params